src/PIL/ImageGrab.py

```diff
@@ -102,7 +102,7 @@ def grabclipboard():
             + filepath
             + '" with write permission)',
             "try",
-            "    write (the clipboard as JPEG picture) to theFile",
+            "    write (the clipboard as «class PNGf») to theFile",
```

Comment:
Please could you explain the `«class PNGf»` syntax?

Reply:
I'm not sure—I'm very far from an osascript expert. It seems that this class of image format doesn't have a proper name in the language, e.g. see: https://apple.stackexchange.com/a/375353/53910

I've tried using `PNG picture`, but it certainly doesn't work.

Reply:
If you create a screenshot on the clipboard with Command-Control-Shift-3, and then run `clipboard info` in AppleScript, here is the result - a list that includes 'JPEG picture' and '«class PNGf»'.
![Screenshot 2023-06-18 at 1 58 21 pm](https://github.com/python-pillow/Pillow/assets/3112309/e3a79d51-ca40-456c-87a6-3b060e65a383)